orchestrator/careplanservice/handle_getquestionnaire.go

```diff
@@ -0,0 +1,21 @@
+package careplanservice
+
+import (
+	"context"
+	fhirclient "github.com/SanteonNL/go-fhir-client"
+	"github.com/SanteonNL/orca/orchestrator/lib/auth"
+	"github.com/zorgbijjou/golang-fhir-models/fhir-models/fhir"
+)
+
+// handleGetQuestionnaire fetches the requested Questionnaire and validates if the requester is authenticated
+func (s *Service) handleGetQuestionnaire(ctx context.Context, id string, headers *fhirclient.Headers) (*fhir.Questionnaire, error) {
+	// Verify requester is authenticated
```

Comment:
I don't think you need an additional check here, since it's handled by the middleware in `service.go` (`auth.Secured` or something)

Reply:
Ah yes it is. I'll remove the check from the methods that don't explicitly need the principal